resources/app.yaml

```diff
@@ -22,7 +22,25 @@ hooks:
               - name: hook
                 image: quay.io/gravitational/debian-tall:0.0.1
                 command: ["/usr/local/bin/kubectl", "apply", "-f", "/var/lib/gravity/resources/resources.yaml"]
-
+  update:
+    spec:
+      apiVersion: batch/v1
+      kind: Job
+      metadata:
+        name: logging-app-update
+      spec:
+        parallelism: 1
+        completions: 1
+        activeDeadlineSeconds: 100
```

Comment:
if the job takes longer than that, `rig freeze` will not get called, i guess.
  * Without knowing what `rig freeze` does (committing a transaction?), how do we know update was successful?   
  * Does `rig status` terminate early upon success?
  * Does it fail at all or simply dumps information to stdout?
  * If the update fails, will `rig freeze` still be called? What is its behavior in either case (success/failure)?